src/initialization/InitDistribution.H

```diff
@@ -98,6 +98,58 @@ namespace impactx::initialization
         amrex::ParticleReal* const AMREX_RESTRICT m_part_py;
         amrex::ParticleReal* const AMREX_RESTRICT m_part_pt;
     };
+
+
+    /** Initialize the input parameters for all distributions that read phase space ellipse parameters from the input
+     *
+     * This function sets the distribution parameters based on the provided phase space inputs.
+     * The parameters include the phase space ellipse intersections for position (`sigmaX`, `sigmaY`), time (`sigmaT`),
+     * momentum (`sigmaPx`, `sigmaPy`) and energy (`sigmaPt`), as well as the correlation terms (`muxpx`, `muypy`, `mutpt`).
+     *
+     * @param pp_dist The parameter parser object.
+     * @param sigx Reference to the variable storing the axis intersection of the phase space ellipse for x position.
+     * @param sigy Reference to the variable storing the axis intersection of the phase space ellipse for y position.
+     * @param sigt Reference to the variable storing the axis intersection of the phase space ellipse for time.
+     * @param sigpx Reference to the variable storing the axis intersection of the phase space ellipse for momentum in x direction.
+     * @param sigpy Reference to the variable storing the axis intersection of the phase space ellipse for momentum in y direction.
+     * @param sigpt Reference to the variable storing the axis intersection of the phase space ellipse for energy.
+     * @param muxpx Reference to the variable storing the mean for momentum in x direction.
+     * @param muypy Reference to the variable storing the mean for momentum in y direction.
+     * @param mutpt Reference to the variable storing the mean for energy.
+     */
+    void
+    setDistributionParametersFromPhaseSpaceInputs (
```

Comment:
Feel free to use `set_distribution_...` spelling for functions/methods in ImpactX. I started a bit inconsistently, but we want to use that going forward. Also makes it nicely the same in Python bindings.